### ISSUE TITLE
Change in game title limits

### ISSUE THREAD

**Issue by shiruva** [isFakeAccount/psnawp#45]
I am currently trying to collect PS4 gameplay time data using the title_stats function, but I get the following error

psnawp_api.core.psnawp_exceptions.PSNAWPBadRequest: {"error":{"reason":"BadRequest","code":3239940,"source":"Gamelist","message":"invalid request, requested limit=500 allowed limit=200","referenceId":"21267330-64b4-1031-a77b-29ec659a0e14"}}

From the message I am thinking that this is due to the game title limit being exceeded, is it possible to change this limit?

**Comment by shiruva** [comment]
thank you so much!

**Comment by isFakeAccount** [comment]
Issue resolved in 1.3.0. Also, in the future if the limit changes, you should be able to change the page size easily. 

```py
title_stat_iterator = user_example.title_stats(limit=10, page_size=10)
```
